Plugin/NodPlugin.h

```diff
@@ -1,117 +0,0 @@
-/* Copyright 2015 Nod Labs */
```

Comment:
yeah, if you look at the new cmake, I added some functionality to grab the dependencies from their install directory. As long as the user has the Nod Configuration installed in its default location it will work. I'm looking into improving it so it either finds it no matter where its installed, or the user uses a -D flag in the cmake to set the location.